frontend/amundsen_application/static/js/components/Lineage/Graph/chart.ts

```diff
@@ -314,8 +345,11 @@ export const buildEdges = (g, targetNode, nodes) => {
     .enter()
     .insert('path', 'g')
     .attr('class', 'graph-link')
-    .attr('d', () => {
-      const o = { x: targetNode.x0 || 0, y: targetNode.y0 || 0 };
+    .attr('d', (d) => {
+      const o =
```

Comment:
d is pretty standard and most will understand what it is, but I don't know what is 'o'. Maybe we can call it 'coordinates' or 'ubication' or something like that?